development-tools/clang.rst

```diff
@@ -54,7 +54,7 @@ A complete list of sanitizers can be found at `Controlling Code Generation
 
 Clang and its sanitizers have strengths (and weaknesses). Its just one tool in
 the war chest to uncovering bugs and improving code quality. Clang should be
-used to complement other methods, including Code Reviews, Valgrind, Coverity,
+used to complement other methods, including Code Reviews, Valgrind,
```

Comment:
Question, do we have more info about "Valgrind"? It was mentioned only once on the entire devguide.

Reply:
How to use valgrind? What is it for?

Reply:
Valgrind is a tool for finding C-level memory issues, like memory leaks or writes to uninitialized memory.

Reply:
https://github.com/python/cpython/blob/main/Misc/README.valgrind

Reply:
Interesting, we also have a https://github.com/python/cpython/blob/main/Misc/README.coverity that perhaps should be cleaned up from the repo.

Reply:
Yeah I found references to coverity on the CPython repo. In addition to README.coverity,
also `Misc\coverity_model.c`

Reply:
> It's documented here: https://github.com/python/cpython/blob/main/Misc/README.valgrind

I updated to link to that doc. Thanks.